backend/main.py

```
@@ -11,208 +17,180 @@
-            yield json.dumps({"status": "progress", "step": 4, "message": "Model finished analysis. Processing image masks..."}) + "\n"
+            yield json.dumps({"status": "progress", "message": "Processing image masks..."}) + "\n"
             await asyncio.sleep(0.1)
 
-            is_glaucoma = False
-            cup_to_disc_ratio = 0.0
-            confidence = 0.0
+            is_glaucoma, cup_to_disc_ratio, confidence = False, 0.0, 0.0
             open_cv_image = np.array(image)
 
             if result is not None:
                 full_img, crops, masks, cdr_val, _, _ = result
```

Ruff (0.15.13)

[warning] Unpacked variable `full_img` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)